src/config/constants.ts

```diff
@@ -125,3 +125,5 @@ export const HOST_VALIDATION_LABELS: { [key in Validation['id']]: string } = {
   'belongs-to-machine-cidr': 'Belongs to machine CIDR',
   'role-defined': 'Role',
 };
+
+export const EVENT_SEVERITIES: Event['severity'][] = ['info', 'warning', 'error', 'critical'];
```

Comment:
```suggestion
export const EVENT_SEVERITIES: Event['severity'][] = ['Info', 'Warning', 'Error', 'Critical'];
```

I'm not sure this is the right place in the code to change this, but we should try to use sentence case throughout the UI.